jobs/JGLOBAL_MARINE_ANALYSIS_FINALIZE

```diff
@@ -1,41 +1,38 @@
 #!/bin/bash
 source "${HOMEgfs}/ush/preamble.sh"
 export WIPE_DATA="NO"
-export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
-source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalpost" -c "base ocnanalpost"
-
+export DATAjob="${DATAROOT}/${RUN}marineanalysis.${PDY:-}${cyc}"
+export DATA="${DATAjob}/marinevariational"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "marineanlfinal" -c "base marineanl marineanlfinal"
 
 ##############################################
 # Set variables used in the script
 ##############################################
-# TODO remove this CDUMP declaration when the GDAS script
-#      exgdas_global_marine_analysis_post.py is updated to look for RUN instead
-#      of CDUMP.
-export CDUMP=${CDUMP:-${RUN:-"gfs"}}
-export CDATE=${CDATE:-${PDY}${cyc}}
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
 export GDUMP=${GDUMP:-"gdas"}
```

Comment:
I doubt, GDUMP is used in a finalize step, but it might be needed in the `__init__`. Just check if that is the case.

Reply:
removed and tested.